tests/format/format_test.py

```diff
@@ -26,7 +26,7 @@ def test_variable_default(var):
     assert f'{var}' == str(var)
     assert f'{var:}' == str(var)
     assert f'{var::}' == str(var)
-    assert '{:}'.format(var) == str(var)
+    assert f'{var}' == str(var)
```

Comment:
I guess this test isn't really needed anymore? Same as line 26.

Reply:
Technically correct, I think. The old test didn't really do anything meaningful either. It exists mainly to check our / my (as the author) understanding of string formatting. I reverted it and suppressed the check.